coding-concepts.js

```diff
@@ -21,26 +21,28 @@ const greeter = (name) => {
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Hello, LEARN Student
+// b) Verify and explain: Returned as "undefined". No value was assigned to name.
 
 
 // --------------------3) What will this log?
 
 var multBy2 = [4, 5, 6, 7, 8].map(value => value * 2)
+
 // console.log(multBy2)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 8, 10, 12, 14, 16
+// b) Verify and explain: Returns : The HOF .map returned value of the array multiplied by 2.
+//
 
 
 // --------------------4) What will this log?
 
 var onlyOdds = [11, 12, 13, 14, 15].filter(value => value % 2 !== 0)
 // console.log(onlyOdds)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 11, 13, 15
```

Comment:
good filter will bring back everything that is true on the condition that is set